src/main/java/org/symphonyoss/symphony/messageml/bi/BiContext.java

```diff
@@ -43,10 +46,23 @@ public void addItem(BiItem item) {
     items.add(item);
   }
 
+  /**
+   * Add a specific item to the context. Used for simple items where we want to keep track of the size/value of the
+   * entity
+   *
+   * @param itemName  name of the item to be added
+   * @param itemValue value to be assigned
+   */
+  public void addItemWithValue(String itemName, Object itemValue) {
```

Comment:
where is it used this method? I can only see it in unit tests.
It is specific for "count" entry, maybe we can make it take the key and values in parameters, I implemented a similar one at Element class level in my PR, I could maybe use this one instead if you keep it. 
here : https://github.com/symphonyoss/messageml-utils/pull/269/files#diff-9b05c5a841bffb66a34b90d0d24ec31bf733fe3041f7cc3ff66a1bb3702255bfR155